app/services/http_client.rb

```diff
@@ -20,12 +20,20 @@ def connection
   end
 
   # FIXME: to use Faraday for file uploads
-  def http_party(endpoint, params)
-    HTTParty.patch(
-      "#{api_base_url}#{endpoint}",
-      headers: { Authorization: authorization_header },
-      body: file_params_body(params)
-    )
+  def http_party(endpoint, method, params)
+    if method.to_s == "post"
+      HTTParty.post(
+        "#{api_base_url}#{endpoint}",
+        headers: { Authorization: authorization_header },
+        body: params.merge(file_params_body(params))
+      )
+    else
+      HTTParty.patch(
+        "#{api_base_url}#{endpoint}",
+        headers: { Authorization: authorization_header },
+        body: file_params_body(params)
+      )
+    end
```

Comment:
We could do this all in one call, I think:
```suggestion
  def http_party(endpoint, method, params)
    HTTParty.send(method
      "#{api_base_url}#{endpoint}",
      headers: { Authorization: authorization_header },
      body: method == :post ? params.merge(file_params_body(params)) : file_params_body(params)
    )
```

Or if not then I think we still don't need to convert the symbol to a string

```suggestion
  def http_party(endpoint, method, params)
    if method == :post
      HTTParty.post(
        "#{api_base_url}#{endpoint}",
        headers: { Authorization: authorization_header },
        body: params.merge(file_params_body(params))
      )
    else
      HTTParty.patch(
        "#{api_base_url}#{endpoint}",
        headers: { Authorization: authorization_header },
        body: file_params_body(params)
      )
    end
```